src/controller/tests/data_model/TestRead.cpp

```diff
@@ -2633,7 +2634,8 @@ void TestReadInteraction::TestReadHandler_MultipleSubscriptionsWithDataVersionFi
         NL_TEST_ASSERT(apSuite,
                        Controller::SubscribeAttribute<TestCluster::Attributes::ListStructOctetString::TypeInfo>(
                            &ctx.GetExchangeManager(), sessionHandle, kTestEndpointId, onSuccessCb, onFailureCb, 0, 10,
-                           onSubscriptionEstablishedCb, nullptr, false, true, dataVersion) == CHIP_NO_ERROR);
+                           onSubscriptionEstablishedCb, nullptr, false, true, dataVersion,
+                           System::Clock::Milliseconds32(30000)) == CHIP_NO_ERROR);
```

Comment:
Again, please document why this needs a non-default timeout.  This is pretty unexpected.